### DISCUSSION TITLE
I modified the `mysql docker compose` to [Feb 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5148] [Discord Threads]
> 🙋‍♂️ **unkn**

**Comment by o1lab** [comment]
> 🙋‍♂️ **mcnaveen**  
#### Here is my `docker-compose`   
####    
#### ```   
#### version: "2.1"   
#### services:    
####   nocodb:    
####     depends_on:    
####       root_db:    
####         condition: service_healthy   
####     environment:    
####       NC_DB: "mysql2://root_db:3006?u=noco&p=special&d=root_db"   
####     image: "nocodb/nocodb:latest"   
####     ports:    
####       - "3005:8080"   
####     restart: always   
####     volumes:    
####       - "nc_data:/usr/app/data"   
####   root_db:    
####     environment:    
####       MYSQL_DATABASE: root_db   
####       MYSQL_PASSWORD: special   
####       MYSQL_ROOT_PASSWORD: password   
####       MYSQL_USER: noco   
####     healthcheck:    
####       retries: 10   
####       test:    
####         - CMD   
####         - mysqladmin   
####         - ping   
####         - "-h"   
####         - localhost   
####       timeout: 20s   
####     image: "mysql:5.7"   
####     restart: always   
####     volumes:    
####       - "db_data:/var/lib/mysql"   
####     ports:   
####       - "3006:3306"   
#### #    below line shows how to change charset and collation   
#### #    uncomment it if necessary   
#### #    command: --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci   
#### volumes:    
####   db_data: {}   
####   nc_data: {}   
#### ```  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1062546835543162950)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
